StubExample/StubExample/ViewController.swift

```diff
@@ -10,17 +10,15 @@ import UIKit
 import Metal
 import MetalPerformanceShadersStub
 
+
 class ViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
+
         if let device = MTLCreateSystemDefaultDevice() {
             let _ = MPSImage.init(device: device, imageDescriptor: MPSImageDescriptor())
         }
-
-        MPSMatrixDescriptor.rowBytes(fromColumns: 0, dataType: .float32) // Crashes always because it uses just the Stub
-
-        var _: CVMetalTextureCache
+//        MPSMatrixDescriptor.rowBytes(fromColumns: 0, dataType: .float32) // Crashes always because it uses just the Stub
```

Comment:
I think you should undo this 2-line change.

Reply:
And change "Crashes always (...)" to "It's ok that it crashes always (...)"

Reply:
it's hard to get obj-c exceptions from swift code. At least swift error handling does not catch such exceptions. I would like to change the code so we validate the method actually throws an exception but it should pass the tests.  

I had a trouble when running Example app tests because somehow the app is launched.